src/main/java/roomescape/exception/NotExistsException.java

```diff
@@ -0,0 +1,18 @@
+package roomescape.exception;
+
+public class NotExistsException extends RuntimeException{
```

Comment:
```suggestion
public class NotExistsException extends RuntimeException {
```

소소한 디테일이지만, [이전 단계에서 얘기했던 컨벤션 내용](https://github.com/next-step/spring-roomescape-reservation/pull/56#discussion_r1629354186)이라 한 번 더 코멘트 달아봅니다!
intelliJ 기능 중 코드를 원하는 컨벤션에 맞게 수정해주는 기능이 있는데, 한 번 알아보고 사용해봐도 좋을 것 같아요.
이런건 누구나 못 보고 넘어갈 수 있으니까요 ㅋㅋㅋ

(+) 아직 space indent 가 아닌 tab indent 네요!

Reply:
제가 깜빡하고 적용을 못 했었네요 ㅎㅎ 
유근님이 추천해주신 spring-javaformat 플러그인도 적용해보고 싶었는데
우선 https://github.com/google/styleguide 활용해서 구글 컨벤션 적용해보았습니다!